packages/core/src/components/forms/_input-group.scss

```diff
@@ -25,16 +25,19 @@ Markup:
 :disabled - Disabled input. Must be added separately to the <code>&#60;input&#62;</code> and <code>&#60;button&#62;</code>. Also add <code>.#{$ns}-disabled</code> to <code>.#{$ns}-input-group</code> for icon coloring (not shown below).
 .#{$ns}-round - Rounded caps. Button will also be rounded.
 .#{$ns}-large - Large group. Children will adjust size accordingly.
+.#{$ns}-small - Small group. Children will adjust size accordingly.
```

Comment:
please add this entry to the `_input.scss` docs as well (but remove note about children, make it match existing notes).